redbeat/schedulers.py

```diff
@@ -191,11 +191,13 @@ def __init__(self, app, **kwargs):
     def setup_schedule(self):
         # cleanup old static entries
         client = redis(self.app)
-        previous = client.smembers(self.app.conf.REDBEAT_STATICS_KEY)
-        current = set(self.app.conf.CELERYBEAT_SCHEDULE.keys())
-        removed = previous - current
+        previous = set(key.decode('utf-8')
```

Comment:
Doing the decode inline with application logic makes it harder to understand the intent of the code.  I feel we should either use decode_responses or abstract out the decode into another method. Being lazy I'd opt for decode_reponses but am not opposed to doing it explicitly.